tests/net/6lo/src/main.c

```diff
@@ -397,7 +396,7 @@ static struct net_6lo_data test_data_1 = {
 	.ipv6.vtc = 0x60,
 	.ipv6.tcflow = 0x00,
 	.ipv6.flow = 0x00,
-	.ipv6.len = { 0x00, 0x00 },
+	.ipv6.len = htons(0),
```

Comment:
IMHO this is totally useless, 0 is always 0 in every byte order. Well, we can go with this too as this makes no difference here.